.github/workflows/dependencies.yaml

```diff
@@ -0,0 +1,40 @@
+name: Custom Dependabot
+on:
+  schedule:
+  - cron: '0 0 * * *'
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v4
+    - name: AdminConsole
+      run: |
+        export VERSION=`curl https://api.github.com/repos/replicatedhq/kots-helm/tags | jq -r .[].name | grep -v alpha | head -1 | tr -d v`
+        sed -i "/^ADMIN_CONSOLE_CHART_VERSION/c\ADMIN_CONSOLE_CHART_NAME = $VERSION" Makefile
+    - name: Embedded Cluster Operator
+      run: |
+        export VERSION=`curl https://api.github.com/repos/replicatedhq/embedded-cluster-operator/tags | jq -r .[].name | head -1 | tr -d v`
+        sed -i "/^EMBEDDED_OPERATOR_CHART_VERSION/c\EMBEDDED_OPERATOR_CHART_VERSION = $VERSION" Makefile
```

Comment:
I think you should match until the end of the line to prevent results like

```
EMBEDDED_OPERATOR_CHART_VERSION = v1.2.0 = v1.1.0 = v1.0.1 = v1.0.0
```

after it gets run multiple times

Reply:
oh never mind, not a sed command I've used before
neat